history.py

```diff
@@ -0,0 +1,15 @@
+class LastFiveMiddleware
```

Comment:
Is this a template?

Reply:
and don't forget the colon after the class name

Reply:
Nice! Thanks guys. I knew something was wrong there.